pkg/querier/tripperware/query.go

```diff
@@ -36,6 +38,12 @@ var (
 	}.Froze()
 )
 
+const (
+	GzipCompression     string = "gzip"
```

Comment:
Let's define type for compression format we support. I think it would be clearer when using them and validating the config

```
type Compression string

const (
  GzipCompression Compression = "gzip"
  SnappyCompression Compression = "snappy"
  NoneCompression Compression = ""
)
```

Reply:
Same for codec types `json` and `protobuf`.